docs/xml-dictionary.md

```diff
@@ -5,28 +5,352 @@ The dictionaries are stored in an *ad hoc* dictionary format. There
+required child: `<title>`.
+
+### Required attributes
+
+ - `id=""` a machine-readable ID used by `<t>` elements to "cite" which
+   source it comes from. This is similar to `id=""` attributes in HTML.
+   The ID **SHOULD** be a short, ASCII-representable string, containing
+   only alphanumerics (U+0030—U+0039, U+0041–U+005A, U+0061–U+007A) and
+   the hyphen-minus character (U+002D). The ID **MUST NOT** contain any
+   spacing or format characters.
+
+### Required children
+
+ - `<title>`: contains the title of the dictionary source in plain text.
+
+### Example
+
+`<t>` elements will cite this source using the id `CW`:
+
+```xml
+<source id="CW">
+  <title>Cree : Words / nehiýawewin : itwēwina</title>
+</source>
+```
+
+`<t>` elements will cite this source using the id `MD`:
+
+
+```xml
+<source id="MD">
+  <title>Maskwacîs Dictionary</title>
+</source>
+```
+
+
+## `<e>`: dictionary entry
+
+Represents a single "word" in the dictionary. In _itwêwina_, each
+[lemma][] is assumed to be assigned to exactly one `<e>` entry and each
+`<e>` is assigned to exactly one [lemma][]. Since the `<t>` element
+indicates which dictionary source a particular translation/definition
+comes from, a single `<e>` element may contain content from multiple
+dictionary sources.
+
+An element may have more than one `<mg>` meaning, each with one or more
+`<t>` translations.
+
+### Required children
+
+ - exactly one `<lg>` (lemma group) containing exactly one `<l>` lemma
+   and `<lc>` lemma category.
+ - one or more `<mg>` meaning groups.
+
+## `<lg>`: lemma group
+
+A **lemma group** describes a [lemma][]. A lemma group has two required
+children, and one optional child.
+
+### Required children
+
+ - exactly one `<l>` lemma element with a required `pos=""` attribute
+ - exactly one `<lc>` lemma content element
+
+### Optional children
+
+ - one optional `<stem>` element
+
+### Example
+
+This describes "acâhkos", which is an animate noun:
+
+```xml
+<lg>
+  <l pos="N">acâhkos</l>
+  <lc>NA-1</lc>
+  <stem>acâhkos-</stem>
+</lg>
+```
+
+
+## `<l>`: lemma
+
+The text of this entry is the canonical [word form][] of the dictionary
+entry. Look-ups in the dictionary will match for this [word form][]
+exactly. In addition to the text of the entry, the `<l>` lemma element
+has one required attribute: `pos=""`, or the **part-of-speech**. The
+part-of-speech is typically "V" or "N", and certain searches will match
+on this value exactly.
+
+
+## `<lc>`: lemma category
+
+The **lemma category** determines which layout will be be used when
+rendering the paradigm.
+
+In Plains Cree, the format of the **lemma category** is as follows:
 
+### Nouns
+
+`ND?(A|I)-(1|2|3|4|4w|5|[?]|x)`
+
+**N**oun, optionally **D**ependent, **A**nimate or **I**nanimate.
+After the hyphen, the noun class follows.
+
+
+### Verbs
+
+`V{class}-(n|v|1|2|3|4|5)`
+
+**V**erb, followed by its transitivity, and the animacy of its
+dependents. Note, that abbreviations are historical Algonquian
```

Comment:
dependents -> arguments or participants

Arguments are what you categorically need for a predicate, whereas dependents are any words in the context with a relation with the predicate, regardless of whether there obligatory.

E.g. He sits [PP: on a couch]

The PP is a dependent of the predicate 'sits', but not obligatory, as you can say 'He sits.'